pages/create/index.js

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+import { useRouter } from "next/router";
+import Image from "next/image";
+import pets from "@/lib/pet";
+import styled from "styled-components";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export default function Create({ onAddPet }) {
+  const router = useRouter();
+  const [formData, setFormData] = useState({
+    name: "",
+    type: "",
+    image: "/assets/images/pets/book.png",
+  });
+
+  const [imagePath, setImagePath] = useState(formData.image);
+
+  const handleTypeChange = (event) => {
+    const selectedType = event.target.value;
+    const imagePath = `/assets/images/pets/${selectedType}.png`;
+    setFormData({ ...formData, type: selectedType, image: imagePath });
+    setImagePath(imagePath); // filled with local one!
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const newFormData = {
+      ...formData,
+      id: uuidv4(),
+      health: 100,
+      hunger: 100,
+      happiness: 100,
+      energy: 100,
+    };
+    onAddPet(newFormData);
+    router.push("/");
+  };
+
+  return (
+    <div>
+      <button onClick={() => router.push("/")}>Back</button>
```

Comment:
I recommend using [Next.js's `<Link>`](https://nextjs.org/docs/pages/api-reference/components/link) component instead of the imperative routing with the `push()` method. 
The <`Link>` component offers a simpler and more declarative approach to navigation and is the standard way to navigate between pages in Next.js applications.

```suggestion
      <button onClick={() => router.push("/")}>Back</button>
```